gamedata/alldefs_post.lua

```diff
@@ -1373,6 +1373,55 @@ function WeaponDef_Post(name, wDef)
 			end
 		end
 
+		if modOptions.shieldsrework == true then
+			local shieldCollisionExemptions = { --add the name of the weapons (or just the name of the unit followed by _ ) to this table to exempt from shield collision. 
+				'corsilo_', 'armsilo_', 'armthor_empmissile', 'armemp_', 'cortron_', 'corjuno_', 'armjuno_'
+			}
+
+			-- For balance, paralyzers need to do reduced damage to shields, as their raw raw damage is outsized
+			local paralyzerShieldDamageMultiplier = 0.25
+
+			if wDef.shield then
+				wDef.shield.repulser = false
+				wDef.shield.exterior = true
+			end
+
+			if (not wDef.interceptedbyshieldtype) or wDef.interceptedbyshieldtype ~= 1 then
+				for _, exemption in ipairs(shieldCollisionExemptions) do
+					if string.find(name, exemption)then
+						wDef.interceptedbyshieldtype = 4
+						break
+					else
+						wDef.interceptedbyshieldtype = 1
```

Comment:
Eventually these values will get baked into the unit defs, where they will be the raw integers.

Unless we want to change unit defs to use variables, but this will break def baking.